omegaconf/_utils.py

```diff
@@ -465,7 +465,7 @@ def get_dict_key_value_types(ref_type: Any) -> Tuple[Any, Any]:
 
     key_type: Any
     element_type: Any
-    if ref_type is None:
+    if ref_type is None or ref_type == Dict:
```

Comment:
why is this needed for dict and not for list?

Reply:
Because is get_dict_key_value_types(), list has its own get_list_element_type()

Reply:
I know.  I am not seeing any changes to get_list_element_type(). This problem looks like it effected both List and Dict, so how come there is a fix in get_dict_key_value_types() and not in get_list_element_type()?

Reply:
Because the way get_list_element_type() is implemented returns None(which should).